influxdb_migration_1_to_timestream_for_influxdb_guide.md

```diff
@@ -0,0 +1,281 @@
+      | bind-socket                        |                                            |
+      | max-body-size                      |                                            |
+      | max-concurrent-write-limit         |                                            |
+      | max-enqueued-write-limit           |                                            |
+      | enqueued-write-timeout             | http-write-timeout                         |
+      |                                    |                                            |
+      | [**logging**]                      |                                            |
+      | format                             |                                            |
+      | level                              | log-level                                  |
+      | suppress-logo                      |                                            |
+      |                                    |                                            |
+      | [**tls**]                          |                                            |
+      | ciphers                            | tls-strict-ciphers                         |
+      | min-version                        | tls-min-version                            |
+      | max-version                        |                                            |
+
+      </details>
+  
+  2. Apply your 1.x custom settings to the comparable InfluxDB 2.7 settings using `influxd` flags, environment variables, or a 2.7 configuration file. For more information about configuring InfluxDB 2.7, see [Configuration options](https://docs.influxdata.com/influxdb/v2/reference/config-options/).
+
+  3. Restart `influxd`.
+
+## Create DBRP mappings
+
+InfluxDB database and retention policy (DBRP) mappings associate database and retention policy combinations with Timestream for InfluxDB buckets. These mappings allow InfluxDB 1.x clients to successfully query and write to Timestream for InfluxDB buckets while using the 1.x DBRP convention.
+
+For more information about DBRP mapping, see Database and retention policy mapping.
+
+To map a DBRP combination to a Timestream for InfluxDB bucket:
+
+1. Create a bucket
+  
+    Create a bucket in Timestream for InfluxDB. We recommend creating a bucket for each unique 1.x database and retention policy combination using the following naming convention:
+
+    ```shell
+    # Naming convention
+    db-name/rp-name
+
+    # Example
+    telegraf/autogen
+    ```
+
+2. Create a DBRP mapping
+    
+    Use the `influx v1 dbrp create` command to create a DBRP mapping. Provide the following:
+
+      - Database name.
+      - Retention policy name (not retention period).
+      - Bucket ID.
+      - (Optional) `--default` flag if you want the retention policy to be the default retention policy for the specified database.
+
+    ```shell
+    # DB with one RP DB
+
+    influx v1 dbrp create \
```

Comment:
All Influx CLI commands should be tested. The InfluxData guide assumes users are running Influx CLI commands locally, on the InfluxDB 2.7 instance, so their examples don't specify host, org, or token.